### PR TITLE
Autohide project summary description on mouse leave (change map section)

### LINKED ISSUE #11: strip descriptions in "Switch Map" button
Hello,

as an option, is it possible to strip the descriptions in the popup following the "Switch Map"?

I am referring to this:

immagine

### PR DESCRIPTION
**Pull request by volterra79**
Closes: #11 

Below a possible solution when mouse is on hover long description text:

ezgif com-gif-maker

### CODE REVIEW

File: src/components/ProjectsMenu.vue [hunk 1]
```diff
@@ -78,4 +84,12 @@ export default {
...
   }
 };
-</script>
\ No newline at end of file
+</script>
+<style scoped>
+  .project-menu-too-long-description {
+    display: -webkit-box;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;
```
Comment (Raruto) [inline src/components/ProjectsMenu.vue]: @volterra79 Please, we can live even better without these vendor specific rules...

**See:** https://stackoverflow.com/questions/15662578/flexible-box-model-display-flex-box-flexbox

File: src/components/ProjectsMenu.vue [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
-      <div v-for="menuitem in state.menuitems"  :key="menuitem.title" @click="trigger(menuitem)" class="col-xs-12 col-sm-4 project-menu">
+      <div v-for="(menuitem, index) in state.menuitems"  :key="menuitem.title" @click="trigger(menuitem)" class="col-xs-12 col-sm-4 project-menu">
         <div class="project-menu-item-image">
           <img :src="logoSrc(menuitem.thumbnail)" class="img-responsive">
         </div>
         <div class="project-menu-item-content">
           <div class="project-menu-item-text">
             <h4 class="project-menu-item-title">{{ menuitem.title }}</h4>
-            <div v-html="menuitem.description"></div>
+            <div :ref="`description_${index}`" class="project-menu-too-long-description" @mouseover="showFullDescription(index)" @mouseout="hideFullDescription(index)" v-html="menuitem.description"></div>
```
Comment (Raruto) [inline src/components/ProjectsMenu.vue]: @volterra79 Beware that "hover" elements have poor accessibility scores, even on mobile they are not so easy to use ...

File: src/components/ProjectsMenu.vue (hunk 1 above)
Comment (volterra79) [reply src/components/ProjectsMenu.vue]: @Raruto i'll check

Comment (volterra79) [reply src/components/ProjectsMenu.vue]: @Raruto changed. Please check and let me know. Thanks

File: src/components/ProjectsMenu.vue (hunk 2 above)
Comment (volterra79) [reply src/components/ProjectsMenu.vue]: @Raruto ok

**Comment by Raruto** [comment]
Converted to draft because it still requires some more in-depth testing regarding mobile usability

**Pull request closed without merging**